src/components/TitleBar/TitleBar.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import ILocalContainerProps from '../../common/structures/ILocalContainerProps';
+import * as styles from './TitleBar.scss';
+import classnames from 'classnames';
+import { Container } from '../Container';
+
+interface IProps extends ILocalContainerProps {}
+
+export const TitleBar = (props: IProps) => (
+	<Container>
+		<div
+			className={classnames(
+				styles.TitleBar,
+				'TitleBar',
```

Comment:
The idea here is to create an unstyled, globally-scoped selector here (`'TitleBar'`), so it's possible to target the component and apply any edge-case styles. The goal would be to eventually apply this to all our components.